hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFileInfo.java

```diff
@@ -443,22 +430,42 @@ public static boolean isHFile(final String fileName) {
   }
 
   /**
-   * @param path Path to check.
-   * @return True if the path has format of a del file.
+   * Checks if the file is a MOB file
+   * @param path path to a file
+   * @return true, if - yes, false otherwise
    */
-  public static boolean isDelFile(final Path path) {
-    return isDelFile(path.getName());
+  public static boolean isMobFile(final Path path) {
```

Comment:
Just to make sure I understand, we're purposefully going to return `false` for any files in the MOB storage area that are from prior to the distributed mob compaction changes?

Reply:
Yes, this is for new MOB files. Old MOB file files are resolved by isHFile call (subset of HFILE_PATTERN). I checked the code.